frontend/src/api.ts

```diff
@@ -182,7 +182,10 @@ export const apiFetch = (
       return response.json().then(body => Promise.reject({
         body,
         error: new Error('API request returned error response'),
-      }));
+      })).catch(error => {
+        // tslint:disable-next-line:no-console
+        console.error('error parsing response:', error);
+      })
```

Comment:
It may be that it makes more sense to move ``Page`` into ``src/containers`` than to try and make it a true component.